include/GoogleSync/GoogleSync.php

```diff
@@ -1109,25 +1261,31 @@ public function doSync($id)
      *
      * Fills the $users array with users that are configured to sync
      *
-     * @return null
+     * @return int added users
      */
     public function setSyncUsers()
     {
-
         $query = "SELECT id FROM users WHERE deleted = '0'";
         $result = $this->db->query($query);
 
-        require_once('modules/Users/User.php');
+        $counter = 0;
         while ($row = $this->db->fetchByAssoc($result)) {
-            $user = new User();
+            $user = BeanFactory::getBean('Users');
+            if (!$user) {
+                throw new Exception('Unable to get User bean.');
+            }
             $user->retrieve($row['id']);
```

Comment:
you can retrive the User with BeanFactory::getBean() above in one step. OR handle the case when there is no user with this ID.

Reply:
Done. One step